include/dao/dao.js

```diff
@@ -322,8 +322,9 @@ module.exports = function DAOModule(pb) {
                 .skip(offset);
 
             //apply sort order
-            if (options.orderBy) {
-                cursor.sort(options.orderBy);
+            var orderBy = options.order || options.orderBy;
+            if (orderBy) {
+                cursor.sort(orderBy);
```

Comment:
Awesome, one small nit-picky change.  Would you mind updating the documentation for the function to include the ```order``` option and mark the ```orderBy``` option as deprecated?